pkgs/shelf/lib/shelf_io.dart

```diff
@@ -57,6 +57,8 @@ Future<HttpServer> serve(
           backlog: backlog,
           shared: shared,
         ));
+  // See https://webtechsurvey.com/response-header/x-powered-by
+  server.defaultResponseHeaders.add('x-powered-by', 'Dart with package:shelf');
```

Comment:
Hrm. I wonder if we should check to see if this is already added and skip?
Should this be a first writer win or last writer win thing? 🤷

Reply:
It can't have been added at this point - we created the `HttpClient` above.

In a refactoring I had planned though, we don't create an `HttpClient` though, and instead take a `Stream<HttpRequest>` to allow the caller to create the client. That design wouldn't be compatible with this approach, the caller would need to set the default themselves.

Reply:
Err, `HttpServer`

Reply:
ack...